app/routes/offender-manager-utilisation.js

```diff
@@ -1,10 +1,55 @@
 const getUtilisationTable = require('../services/get-utilisation-table')
+const UtilisationDate = require('../services/domain/utilisation-date')
 const ORG_UNIT_TYPE = require('../constants/organisation-unit-type-enum')
+const dateFormatter = require('../services/date-formatter')
+const ValidationError = require('../services/errors/validation-error')
 
 module.exports = function (router) {
   router.get(`/caseload-utilisation/${ORG_UNIT_TYPE.OFFENDER_MANAGER}/:id/`, function (req, res, next) {
-    return res.render('utilisation', {
-      utilisation: getUtilisationTable(ORG_UNIT_TYPE.OFFENDER_MANAGER, req.params.id, req.query.date)
-    })
+
+    try {
+      var fromUtilisationDate, toUtilisationDate
+
+      if(Object.keys(req.query).length === 0) {
+        fromDate = dateFormatter.now().subtract(1, "years")
+        toDate = dateFormatter.now()
+
+        fromUtilisationDate = new UtilisationDate(
+          fromDate.date(),
+          fromDate.month() +1,
+          fromDate.year()
+        )
+        toUtilisationDate = new UtilisationDate(
+          toDate.date(),
+          toDate.month() +1,
```

Comment:
moment has a zero starting index for months which can throw it off a bit (this means 3 === April). I can move this to the object itself but I wasn't sure if hiding this quirk of the moment lib was wise or not. 

Also .day() caught me out too, returning the day of the week. Thats why it's using .date()